YIF.Core.Domain/DtoModels/EntityDTO/InstitutionOfEducationDTO.cs

```diff
@@ -67,7 +67,7 @@ public class InstitutionOfEducationDTO
         /// Is the institution Of Education favorite
         /// </summary>
         public bool IsFavorite { get; set; }
-
```

Comment:
Add swagger comments, please.

Reply:
it's DTO, so i think that we don't need them there at all